Makefile

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
we would use thoth-ci to build this image, so it would directly build from Dockerfile.
Makefile is not needed if it is not doing anything more the build.WDYT?

Reply:
yeah we don't need it, it's just convenient when building the image locally